crates/astria-bridge-withdrawer/src/withdrawer/submitter/mod.rs

```diff
@@ -0,0 +1,286 @@
+use std::{
+    sync::Arc,
+    time::Duration,
+};
+
+use astria_core::protocol::transaction::v1alpha1::{
+    Action,
+    TransactionParams,
+    UnsignedTransaction,
+};
+use astria_eyre::eyre::{
+    self,
+    eyre,
+    Context,
+};
+pub(crate) use builder::Builder;
+use sequencer_client::{
+    tendermint_rpc::endpoint::broadcast::tx_commit,
+    Address,
+    SequencerClientExt as _,
+    SignedTransaction,
+};
+use signer::SequencerSigner;
+use state::State;
+use tokio::{
+    select,
+    sync::mpsc,
+    time::Instant,
+};
+use tokio_util::sync::CancellationToken;
+use tracing::{
+    debug,
+    error,
+    info,
+    info_span,
+    instrument,
+    warn,
+    Instrument as _,
+    Span,
+};
+
+use super::{
+    batch::Batch,
+    state,
+};
+
+mod builder;
+mod signer;
+
+pub(super) struct Handle {
+    pub(super) batches_tx: mpsc::Sender<Batch>,
```

Comment:
Make this private. No point having a handle if it just reveals its contents. Also, please don't overload abbreviations. In other parts of this crate `_tx` is used as `transaction`, whereas here it's used as `sending`